ngraph/core/src/op/convert.cpp

```diff
@@ -119,15 +119,18 @@ namespace convert
             NGRAPH_TYPE_CASE(evaluate_convert, u1, arg, out);
             NGRAPH_TYPE_CASE(evaluate_convert, u4, arg, out);
             NGRAPH_TYPE_CASE(evaluate_convert, u8, arg, out);
+            NGRAPH_TYPE_CASE(evaluate_convert, u16, arg, out);
+            NGRAPH_TYPE_CASE(evaluate_convert, u32, arg, out);
+            NGRAPH_TYPE_CASE(evaluate_convert, u64, arg, out);
             NGRAPH_TYPE_CASE(evaluate_convert, i4, arg, out);
             NGRAPH_TYPE_CASE(evaluate_convert, i8, arg, out);
-            NGRAPH_TYPE_CASE(evaluate_convert, i32, arg, out);
             NGRAPH_TYPE_CASE(evaluate_convert, i16, arg, out);
+            NGRAPH_TYPE_CASE(evaluate_convert, i32, arg, out);
             NGRAPH_TYPE_CASE(evaluate_convert, i64, arg, out);
-            NGRAPH_TYPE_CASE(evaluate_convert, u32, arg, out);
-            NGRAPH_TYPE_CASE(evaluate_convert, u64, arg, out);
+            NGRAPH_TYPE_CASE(evaluate_convert, bf16, arg, out);
             NGRAPH_TYPE_CASE(evaluate_convert, f16, arg, out);
             NGRAPH_TYPE_CASE(evaluate_convert, f32, arg, out);
+            NGRAPH_TYPE_CASE(evaluate_convert, f64, arg, out);
```

Comment:
I think we should not have `f64` data type, OV doesn't support this data type.

Reply:
Removed [here](https://github.com/openvinotoolkit/openvino/pull/5745/commits/62cee14aae29c7882e3018d0062e1a1ef6d468bc).